api/v1alpha1/common_types.go

```diff
@@ -62,7 +62,7 @@ type FilterByNeutronTags struct {
 }
 
 // +kubebuilder:validation:Enum:=4;6
-type IPVersion int8
+type IPVersion int32
```

Comment:
@mandre It's fine, but I'm curious to know why.

Reply:
Upstream API conventions dictate that you should use int32 or int64 for all integer types, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types